app/assets/stylesheets/response_view.sass

```diff
@@ -27,6 +34,17 @@
     .fa-warning
       color: darken($screensmart-red, 2%)
       padding-right: 0.67rem
+  .results-table
+    border: 2px solid $response-estimate-interpretation-border-color
+    background-color: $response-background-color
+    padding: 0.5rem 1.6rem
+    border-radius: $response-border-radius
```

Comment:
Ik vind de naam wel prima zo, het is de border die gebruikt wordt bij het tonen van de response, dat dekt de lading prima.